python/semantic_kernel/connectors/ai/mistral_ai/prompt_execution_settings/mistral_ai_prompt_execution_settings.py

```diff
@@ -28,11 +35,41 @@ class MistralAIChatPromptExecutionSettings(MistralAIPromptExecutionSettings):
     temperature: float | None = Field(None, ge=0.0, le=2.0)
     top_p: float | None = Field(None, ge=0.0, le=1.0)
     random_seed: int | None = None
+    function_call_behavior: FunctionCallBehavior | None = Field(None, exclude=True)
```

Comment:
Is this necessary? `function_choice_behavior` already lives in the base class `PromptExecutionSettings`, and there is no need to provide backward compatibility.

Reply:
I agree -- we want to move away from FunctionCallBehavior and should please not introduce it into new classes. :)